### PR TITLE
sink: refine mysql log, remove redundant SQL count log

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

sink prints too much logs in each exec DML loop

### What is changed and how it works?

- remove log for each time that MySQL sink executes DMLs, instead of a periodically print status

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cdc/sink/mysql.go [hunk 1]
```diff
@@ -345,6 +353,34 @@ func (s *mysqlSink) Close() error {
 	return nil
 }
 
+func (s *mysqlSink) PrintStatus(ctx context.Context) error {
+	changefeedID := util.ChangefeedIDFromCtx(ctx)
```
Comment (leoppro) [inline cdc/sink/mysql.go]: if the caller of sink is owner, could we get changefeedId from context?

how about store a value in context, in order to get the role of caller?

Comment (leoppro) [reply cdc/sink/mysql.go]: by the way, maybe we can storage `changefeedId` in `mysqlSink` struct to avoid query changedfeedId  continually

Comment (amyangfei) [reply cdc/sink/mysql.go]: Add more options in `sink opt`, as `changefeedID` and `captureID` are used quite often. The `role` is not put into sink now, as we only call `PrintStatus` of the sink in processor. We can add the role information if we really need it.

**Pull request merged**